contracts/diamond/Diamond.sol

```diff
@@ -50,6 +65,53 @@ contract Diamond is DiamondStorage {
         }
     }
 
+    receive() external payable virtual {}
```

Comment:
The absence of receive was intentional. If there is a compiler warning, please figure out how to disable it :)